Cargo.toml

```diff
@@ -3,9 +3,16 @@ name = "appcheck-backend"
 version = "0.1.0"
 edition = "2021"
 
+[[bin]]
+name = "jwt-bearer"
+path = "src/main.rs"
+
 [dependencies]
 axum = ">= 0.7"
 base64 = ">= 0.22"
+bs58 = "0"
+clap = { version = "4", features = ["derive"] }
```

Comment:
sort of annoying to have a library that has a main function with clap deps.. Perhaps make separate crates in a workspace to clean out the deps?